grizli/multifit.py

```diff
@@ -1411,7 +1412,8 @@ def __init__(self, beams, group_name=None, fcontam=0., psf=False, polyx=[0.3, 2.
 
         if isinstance(beams, str):
             self.load_master_fits(beams, verbose=verbose)
-
+            # check if isJWST
+            isJWST = prep.check_isJWST(beams)
```

Comment:
Also appears in the loop just below

Reply:
It has to check differently for the second case (it checks beams if beams is a string and beams[0] if beams is a list of strings)